readthedocs/builds/migrations/0031_migrate_version_data_to_build.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.17 on 2020-11-25 15:56
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    """Migrate all data from versions to builds."""
+    Build = apps.get_model('builds', 'Build')
+    for build in Build.objects.all():
+        # When the build is saved, the fields are updated.
+        build.save()
```

Comment:
I'm worried about this migration... I think is going to take a while, and maybe block the table? We could also do this incrementally, this is by overriding the delete method of the version (or use the pre-signal) and update all of their builds before that.

Reply:
I don't believe it will block. It's basically just running the Python code -- but we should definitely be using an `iterator()` here, or finding another clean way of handling this without eating a ton of memory.

Reply:
updated! Also, we can run 0031 and 0032 after the deploy, so we don't need to wait so long before deploying the webs.

Reply:
I have added that note in the release board